src/adhocracy_mercator/adhocracy_mercator/scripts/export_users.py

```diff
@@ -43,27 +88,39 @@ def export_users():
 
     users = query.execute()
 
+    # FIXME: set 100 instead of 0 on prod
```

Comment:
> @pallix @nico I added some styleguide comments

We don't need to refactor this script, but it would be good to make it better next time. 
Also maybe we need to improve the backend documentation about  catalog searches with the pool sheet.